### PR TITLE
Fix depwarn on 0.5 and 0.6

### PR DESCRIPTION
**Pull request by yuyichao**

### CODE REVIEW

**Comment by tlnagy** [comment]
The only reason this seems to have failed is due to a stalled build, could you restart the build and merge? A package I'm developing depends on Compose and the tests are failing on nightly. This PR should fix that.

**Comment by tlnagy** [comment]
It seems like everything is green. It would be great to have this merged and a new version tagged as soon as possible. Thanks!

**Comment by tlnagy** [comment]
Could someone with commit access to this repo merge this so that Gadfly doesn't crash on v0.5? Pinging @timholy @shashi @aviks @tkelman

File: test/runtests.jl [hunk 1]
```diff
@@ -10,8 +10,18 @@ cd(testdir)
 exampledir = joinpath(testdir, "..", "examples")
 for ex in readdir(exampledir)
     endswith(ex, ".jl") || continue
-    srand(1) #Needed so that SVG uuid is reproducible
-    include(joinpath(exampledir, ex))
+    file = joinpath(exampledir, ex)
+    expr = quote
+        module $(Symbol(replace(ex, ".", "_")))
```
Comment (tkelman) [inline test/runtests.jl]: why was this needed?

Comment (yuyichao) [inline test/runtests.jl]: Method overwrite warnings between tests.
